lib/config/core.js

```diff
@@ -76,6 +76,7 @@ function load () {
   cb = once(function (er, conf) {
     if (!er)
       exports.loaded = conf
+      loading = false
```

Comment:
@charmander Ah, you are right. Fixed in https://github.com/KenanY/npm/commit/fa27d2488406f7378dbb2e9f9b1b3cfef86ce1f2 and force-pushed